### ISSUE TITLE
Mouse movement is very fast

### ISSUE THREAD

**Issue by jeancroy** [Dygmalab/Bazecor#166]
Intentionally reuse wording of #57

It should be configurable.

It's very hard for me to press the key for a short enough time so it travel at least like 30 pixels per press. The mouse keyboard is unusable for anything precision. Coincidentally if I use the on keyboard mouse, it's because I work with thing the size of a character, so precision is important.
If I really need something on the other side of the screen, then I use real mouse.

Others like #149 would like mouse key to work at macro level then use real mouse as micro adjustment.

In conclusion it's very hard to find a single setting that make people happy.

**Comment by AlexDygma** [comment]
Totally agreee @jeancroy 

now we have settings for mouse parameters on the preferences section!
